utility/src/main/java/org/oppia/android/util/parser/html/HtmlParser.kt

```diff
@@ -240,6 +245,29 @@ class HtmlParser private constructor(
       )
     }
 
+    /**
+     * Returns a new [HtmlParser] with the empty entity type and ID for loading images,
+     * doesn't require GCS properties and imageCenterAlign set to false
+     * optionally specified [CustomOppiaTagActionListener] for handling custom Oppia tag events.
+     */
+    fun create(
+      displayLocale: OppiaLocale.DisplayLocale
+    ): HtmlParser {
+      return HtmlParser(
+        context,
+        urlImageParserFactory,
+        gcsResourceName = "",
+        entityType = "",
+        entityId = "",
+        imageCenterAlign = false,
+        consoleLogger,
+        cacheLatexRendering = enableCacheLatexRendering.value,
+        customOppiaTagActionListener = null,
+        null,
```

Comment:
This should be a named variable like the rest. Do avoid mixing named and unnamed variables in the same function.

Reply:
Hi Adhiambo, thanks for reviewing! Is this null variable the only unnamed variable? For others like consoleLogger, should we replace it as consoleLogger = consoleLogger?

Reply:
It is adviasable to avoid mixing named an unnamed variables. So in this case, we should also replace the rest like you've metioned, e.g `consoleLogger = consoleLogger`